src/clients/FilterClient.js

```diff
@@ -0,0 +1,18 @@
+export function updateFilterOptions(filter) {
+  var url = filter.url;
+
+  $.ajax({
+    url: url,
+    type: "GET",
+    dataType: "json",
+    success: function(data) {
+      filter.options = data;
+      filter.value = filter.value || filter.options[0].value;
+    },
+    error: function(xhr, status, error) {
+      console.error(xhr);
```

Comment:
I've just removed it for the moment. I can add it back in if this gets passed back for bug-fixing. (when?)